src/routes/AddEditQuiz/index.tsx

```diff
@@ -69,139 +58,139 @@ export default function AddEditQuiz() {
         body: `There ${
           unSavedQuestionsCount > 1 ? `are ${unSavedQuestionsCount} empty questions` : 'is an empty question'
```

Comment:
Should we create a plural like function?

Reply:
fixed